modular_skyrat/modules/goofsec/code/terragov.dm

```diff
@@ -452,6 +495,7 @@ GLOBAL_LIST_INIT(call911_do_and_do_not, list(
 	back = /obj/item/storage/backpack
 	uniform = /obj/item/clothing/under/sol_peacekeeper
 	shoes = /obj/item/clothing/shoes/jackboots
+	gloves = /obj/item/clothing/gloves/tackler/combat
```

Comment:
this is the guriella glove yes? they don't hide prints

Reply:
> this is the guriella glove yes? they don't hide prints

that's for SWAT, they aren't here to investigate
marshal gets white gloves, which should hide prints